Core/SaveState.cpp

```diff
@@ -364,7 +364,6 @@ namespace SaveState
 		}
 
 		// Memory is a bit tricky when jit is enabled, since there's emuhacks in it.
-		auto savedReplacements = SaveAndClearReplacements();
```

Comment:
This change looks very bad and probably broke save states?

-[Unknown]

Reply:
Yeah, sigh. Should have reviewed this one more closely.